### ISSUE TITLE
krita: crashes when used with wacom stylus

### ISSUE THREAD

**Issue by petabyteboy** [NixOS/nixpkgs#54470]
## Issue description

When I try to draw something in Krita with the Wacom pen of my X230T, as soon as the stylus is in range, Krita crashes with the following message: `malloc_consolidate(): invalid chunk size`

I also tried using krita from nixos-stable with `nix-shell '<stable>' -A krita --run krita`, but it crashes with the message `double free or corruption (out)` when I try to use the wacom stylus.

### Steps to reproduce

- Open Krita
- Create a new image
- Hover the wacom pen over the drawing area

## Technical details

 - system: `"x86_64-linux"`
 - host os: `Linux 4.19.16, NixOS, 19.03pre166987.bc41317e243 (Koi)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.2`
 - channels(root): `"nixos-19.03pre166987.bc41317e243, stable-18.09.1974.50f41ea2fcf"`
 - channels(pbb): `"nixos-19.03pre166987.bc41317e243, stable-18.09.1974.50f41ea2fcf"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Please run `nix-shell -p nix-info --run "nix-info -m"` and paste the
results.

**Comment by veprbl** [comment]
Have you tried filing this issue upstream https://krita.org/en/get-involved/report-a-bug/ ?

**Comment by petabyteboy** [comment]
> Have you tried filing this issue upstream https://krita.org/en/get-involved/report-a-bug/ ?

No. Because I did not experience this issue when I last used Krita a few days ago on Arch Linux, I thought it might be a packaging issue.

> I've also noticed some issues in this update #52247.
> Were these present in the current version? @petabyteboy
> 
> ###### This techincially isn't issue hijacking

4.1.7 resolves the problem, so it is indeed an issue that needs to be resolved in nixpkgs.

Also I noticed I made a mistake when I tried to test the stable Krita derivation, now that I actually tested 4.1.1 from stable, it works flawlessly. A bugfix for stable will not be necessary.
